var/spack/repos/builtin/packages/lua/package.py

```diff
@@ -12,6 +12,22 @@
 from spack.package import *
 from spack.util.executable import Executable
 
+# https://github.com/guix-mirror/guix/raw/dcaf70897a0bad38a4638a2905aaa3c46b1f1402/gnu/packages/patches/lua-pkgconfig.patch
```

Comment:
Add another line of intro to this, e.g.
```suggestion
# This is the template for a pkgconfig file for rpm
# https://github.com/guix-mirror/guix/raw/dcaf70897a0bad38a4638a2905aaa3c46b1f1402/gnu/packages/patches/lua-pkgconfig.patch
```

Reply:
I'd also start this with an underscore, as an indication this is a private variable.